pandas/io/spss.py

```diff
@@ -0,0 +1,33 @@
+from typing import Union, Sequence
+from pandas.core.api import DataFrame
+
+
+def read_spss(path: str,
+              usecols: Union[str, Sequence[str], None] = None,
```

Comment:
this can be a string?

we need to define a ListLike in pandas._typing I think (doesn't have to be done here)

Reply:
Standard `typing` has `Sequence` and `Iterator` (and `List` for return types), but I agree that a duck type `ListLike` would be useful.